string iio_path # A valid IIO path to a context/device/channel
---
bool success
string message
string[] data
